docs/src/guides/developers/using-component-types.mdx

```diff
@@ -1,4 +1,6 @@
-# How to use component types (Typescript | Flow)
+---
+title: Using component types
```

Comment:
This will expand with known fields, such as whatever we decide to replace the current `excerpt` with. I imagine it'd be good to have an image field for metadata purposes as well.

Reply:
Could you elaborate? Are you talking about two separate things? Is there something I should change here?

Reply:
I think I was just trying to plan for the future. It's also possible to type the frontmatter in Markdown files in Gatsby, so I assumed it'd be good to think about it.

Reply:
I still don't understand, sorry 😅 What do you mean by "type the frontmatter"? I think it will be really good to familiarize myself with the current process, how the documents are currently written so that I can replicate the useful parts of the process and discard or improve the rest.

Reply:
When there are things like images in frontmatter, it can help to explicitly add that to the Gatsby schema. Like https://gitlab.skypicker.com/kiwi/handbook/-/blob/master/gatsby-node.js#L225

Reply:
Oh, cool! I'll definitely add that. 👍